.azure-pipelines/all.yml

```diff
@@ -61,6 +61,8 @@ jobs:
       os: linux
     - checkName: exim
       displayName: exim
+    - checkName: fiddler
+      displayName: fiddler
```

Comment:
After rebasing your `master` branch with the upstream `master` branch, these should be the only changes on this file. 

How to do a clean rebase:

```
git fetch upstream
git checkout master
git rebase upstream/master   
# 1) fix any conflicts between HEAD and upstream/master
# 2) git add .
# 3) git rebase --continue
# repeat steps 1-3 until all commits are successfully rebased
git push origin master --force-with-lease
```